controller/templates/confd_settings.py

```diff
@@ -46,3 +46,14 @@
 {{ end }}
 
 UNIT_HOSTNAME = '{{ or (.deis_controller_unitHostname) "default" }}'
+
+# AUTH
+# LDAP
+LDAP_ENDPOINT = '{{ .deis_controller_auth_ldap_endpoint }}'
```

Comment:
weird, I created a deis cluster without the keys and everything works fine, but indeed the confd will wait for the keys forever.

@bacongobbler You have a suggestion for these conditional? 
I imagine something like "controller/auth/ldap/enabled" with a false by default. Its enough ?